devfile.yaml

```diff
@@ -0,0 +1,33 @@
+schemaVersion: 2.1.0
+metadata:
+  name: java-lombok
+attributes:
+  che-theia.eclipse.org/sidecar-policy: USE_DEV_CONTAINER
```

Comment:
I think we can get rid of it now as it's the default

Reply:
Thanks, done